spec/ics-027-direct-interchain-exchange/ReadMe.md

```diff
@@ -0,0 +1,74 @@
+# Cosmos Direct Interchain Exchange Protocol (DIEP)
```

Comment:
if only there was a way we could make this acronym DIPER... `Direct Interchain Protocol for Exchange and Relay` 👍 👶

Reply:
Interesting, this is semantics, but we should probably design the protocol in a versatile enough way such that directly or indirectly (through the hub) tokens may be exchanged using the protocol.

Reply:
> Two cents: is it fare to call a protocol **direct** in the presence of a Hub (- 3rd party)?

I agree. It is relatively indirect than original solution via direct IBC connection between those two chains.

In this context, it is more like "Cosmos Centralized Interchain Exchange Protocol". People might hate the word "centralized" but it is what this doc proposing I think.